rocAL/rocAL/include/readers/image/reader.h

```diff
@@ -68,6 +77,7 @@ struct ReaderConfig
     void set_sequence_length(unsigned sequence_length) { _sequence_length = sequence_length; }
     void set_frame_step(unsigned step) { _step = step; }
     void set_frame_stride(unsigned stride) { _stride = stride; }
+    void set_mode(FileMode mode) { _mode = mode; }
```

Comment:
please rename this api call to set_file_mode instead of set_mode
also _mode can also be renamed to _file_mode